src/Mapping/Operator/Factory/AsStructuredTable.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\DataImporterBundle\Mapping\Operator\Factory;
+
+use Pimcore\Bundle\DataImporterBundle\Mapping\Operator\AbstractOperator;
+use Pimcore\Bundle\DataImporterBundle\Mapping\Type\TransformationDataTypeService;
+use Pimcore\Model\DataObject\Data\StructuredTable;
+
+class AsStructuredTable extends AbstractOperator
+{
+    /**
+     * @param mixed $inputData
+     * @param bool $dryRun
+     *
+     * @return StructuredTable|string
+     */
+    public function process($inputData, bool $dryRun = false)
+    {
+        if (!is_array($inputData)) {
+            $inputData = [$inputData];
+        }
+        if ($dryRun) {
+            if (empty($inputData)) {
+                return "";
+            }
+            return (new StructuredTable($inputData))->getHtmlTable(
```

Comment:
Why return a html table here? 
Shouldn't be that in `generateResultPreview`, as for example [here](https://github.com/pimcore/data-importer/blob/43fb4128dac8284fcfc29b7ec6d2ffed99c4d22b/src/Mapping/Operator/Factory/Gallery.php#L73)